adhocracy-plus/templates/a4dashboard/includes/progress_line.html

```diff
@@ -0,0 +1,79 @@
+{% load i18n a4dashboard_tags a4_candy_dashboard_tags rules static %}
+
+<ul class="progress-line u-spacer-bottom">
+    <li class="progress-line__entry">
+        <span class="progress-line__line--bottom"></span>
+        <i class="fas fa-check-circle u-success progress-line__icon" title="{% trans 'Project type created' %}" aria-label="{% trans 'Project type created' %}"></i>
+        <span>{% trans 'Create project' %}</span>
+    </li>
+
+    <li class="progress-line__entry">
+        <span class="progress-line__line--top"></span>
+        <span class="progress-line__line--bottom"></span>
+        {% if project_progress.project_is_complete %}
+            <i class="fas fa-check-circle u-success progress-line__icon" title="{% trans 'Project type ready for publication' %}" aria-label="{% trans 'Project type ready for publication' %}"></i>
```

Comment:
We use the "project type" in mB as it can be normal projects, linkages or bplans. Here "Project ready for publication" would be better.